src/main/java/nl/tudelft/semgroup4/GameEndedState.java

```diff
@@ -22,208 +20,406 @@
         textFieldPlayer1.setConsumeEvents(true);
         textFieldPlayer1.setMaxLength(10);
-        
-        textFieldPlayer2 = new TextField(container,
-                typeFontPlayer,
-                container.getWidth() / 5 * 4,
-                container.getHeight() / 2,
-                200,
+
+        textFieldPlayer2 = new TextField(container, typeFontPlayer,
+                container.getWidth() / 5 * 4, container.getHeight() / 2, 200,
                 50);
         textFieldPlayer2.setConsumeEvents(true);
         textFieldPlayer2.setMaxLength(10);
-        
-        mouseOverTextField1 = new MouseOverArea(container,
-                null,
-                container.getWidth() / 5 * 4,
+    }
+
+    /**
+     * Draws the first players information onto the screen.
+     * 
+     * @param container
+     *            container in which to draw onto.
+     */
+    public void drawFirstPlayerInfo(GameContainer container) {
+        // draws all information of the fist player
+        typeFontPlayer.drawString(container.getWidth() / 8,
+                container.getHeight() / 3, "PLAYER 1", Color.white);
+        typeFontPlayer.drawString(container.getWidth() / 5 * 2,
                 container.getHeight() / 3,
-                textFieldPlayer1.getWidth(),
-                textFieldPlayer1.getHeight());
-        
-        mouseOverTextField2 = new MouseOverArea(container,
-                null,
-                container.getWidth() / 5 * 4,
-                container.getHeight() / 2,
-                textFieldPlayer2.getWidth(),
-                textFieldPlayer2.getHeight());
-        
-        mouseOverContinueButton = new MouseOverArea(container,
-                null,
-                container.getWidth() - margin - typeFontPlayer.getWidth(gameContinueText),
-                container.getHeight() - margin - typeFontPlayer.getHeight(),
-                typeFontPlayer.getWidth(gameContinueText),
-                typeFontPlayer.getHeight());
-        
+                Integer.toString(players[0].getScore()), Color.white);
+        typeFontPlayer.drawString(container.getWidth() / 5 * 3,
+                container.getHeight() / 3,
+                Integer.toString(players[0].getMoney()), Color.white);
+
     }
-    
+
     /**
-     * Setup for initializing extra data after the state has been made at the beginning
-     * of the app.
-     * @param players Players currently in the game
-     * @param won True if the game has been won
+     * draws the seconds player information onto the screen.
+     * 
+     * @param container
+     *            container in which to draw the information
+     * @param graphics
+     *            graphics to use to draw the elements
      */
-    public void setup(Player[] players, boolean won) {
-        this.won = won;
-        this.players = players;
+    public void drawSecondPlayerInfo(GameContainer container, Graphics graphics) {
+        typeFontPlayer.drawString(container.getWidth() / 8,
```

Comment:
Couldn't you use a for loop where you loop over every player to draw its information. You could use the index you are currently in to determine the location of the items you need to place. This would save quite some code I think.